packaging/windows/build_pypi_pkg_download_proxy/setup.py

```diff
@@ -14,18 +14,23 @@
 
 install_requires = [
    'pip',
+   'future',
+   'numpy',
+   'typing_extensions',
+   'dataclasses; python_version < "3.7"'
```

Comment:
Why any of those packages are needed for the proxy?

Reply:
The wheel is downloaded with no-deps flag. On a clean Python environment, without these dependencies, the installation would fail

Reply:
Would it be possible to download with dependencies? Having extra dependencies to maintain here seems like it'd be a headache later on down the line.

Reply:
As far as I can see, we are only doing this for the 1.7.x release. By the time 1.8.0 is out, the cudatoolkit should be ready on Windows and we won’t need this proxy package anymore. @nbcsm Can you confirm this?